syntax/ocaml.vim

```diff
@@ -6,6 +6,7 @@
 "               Issac Trotts      <ijtrotts@ucdavis.edu>
 " URL:          https://github.com/ocaml/vim-ocaml
 " Last Change:
+"               2022 Jul 20 - Improved highlighting of type decl (Jules Aguillon)
```

Comment:
As a side note, the previous date: 2019 isn't really correct.

Reply:
It seems to really be authored in 2019 but committed in 2022: 41fb0d069ccdb52a305a5df5b515ff43bb4c925e

Reply:
Agreed with @rgrinberg. I did most of the work back in 2019 (not everything though: the recent polishing work got squashed into that commit, which kept the earlier date), and I thought it was relevant to mention that date (because of the evolution of the language), *but* I realize now that people would rather be interested in the release date.

Reply:
I added a commit for the last two changes.